Core/Core/Models/Assignment.swift

```diff
@@ -209,6 +212,74 @@ extension Assignment {
             return .unlocked
         }
     }
+
+    public var submissionStatus: String? {
```

Comment:
Looks like you're always returning a non-optional `String`. Is there a difference between returning `""` vs `nil` here?